ktor-core/test/org/jetbrains/ktor/tests/http/CookiesTest.kt

```diff
@@ -0,0 +1,118 @@
+package org.jetbrains.ktor.tests.http
+
+import org.jetbrains.ktor.application.*
+import org.jetbrains.ktor.http.*
+import org.jetbrains.ktor.testing.TestApplicationHost
+import org.jetbrains.ktor.testing.TestApplicationRequest
+import org.jetbrains.ktor.tests.withTestApplication
+import java.text.SimpleDateFormat
+import kotlin.test.assertEquals
+import org.junit.Test as test
+
+class CookiesTest {
+    test fun `simple cookies`() {
+        withTestApplication {
```

Comment:
`TestApplicationRequest` requires `Application`. Alternatively I can extract parse/render implementations to some public functions but these public functions becomes visible to everyone